STEERING-COMMITTEE.md

```diff
@@ -36,8 +36,11 @@ the way we run this committee, based on feedback from the community.
 1. Steward, control access, delegate access, and establishes processes
    regarding, all Knative project resources and has the final say in the
    disposition of those resources.
-1. Manage the Knative brand and decide which things can be called "Knative" and
-   how that mark can be used in relation to other efforts or vendors.
+1. Advise the Trademark Committee on usage of the brand and marks in relation to
+   promoting the community. This advice is non-binding on the Trademark
+   Committee.
```

Comment:
I'm curious about this means for the committee hierarchy.  What's the 'root node' in this tree? the KSC or the KTC?  (please consider this question in tandem with my question regarding practical conformance suite example below)